TaskList/app/views/layouts/_person_printout.html.erb

```diff
@@ -0,0 +1,20 @@
+<table class="table table-hover">
```

Comment:
This and your other templates in the `layouts/` directory seem to be closer to partials. Is there a particular reason that you placed them here?